Makefile

```diff
@@ -116,4 +116,23 @@ local-attack-scenario-stop:
 
 .PHONY: local-attack-scenario-restart
 local-attack-scenario-restart:
-	docker-compose -f deploy/docker-compose/docker-compose-fixed-instrumented-attack.yml restart
\ No newline at end of file
+	docker-compose -f deploy/docker-compose/docker-compose-fixed-instrumented-attack.yml restart
+
+.PHONY: local-baseline-start
```

Comment:
You caught some other changes in the PR. Just a heads-up if it was accidental.

Reply:
Ah sorry, I should have mentioned that in the PR notes. Yes this was intentional. Running `DD_API_KEY=[key] make local-baseline-start` and `make local-baseline-stop` is a cleaner way to build / teardown all containers locally. I use this for local dev